pkg/kubelet/kubelet.go

```diff
@@ -576,6 +576,15 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 			if err := server.Start(); err != nil {
 				return nil, err
 			}
+
+			// Create dockerLegacyService when the logging driver is not supported.
+			supported, err := dockershim.IsCRISupportedLogDriver(klet.dockerClient)
+			if err != nil {
+				return nil, err
+			}
+			if !supported {
+				klet.dockerLegacyService = dockershim.NewDockerLegacyService(klet.dockerClient)
```

Comment:
Can we also log an information on which logging driver is using as part of NodeStatus? Today we don't have any such information.